elasticsearch/reindex/progress.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Usage: [watch -t] ./progress.sh <current index doc count>
+
+DOC_COUNT=$1
```

Comment:
Seems the % completion of this step is more useful so automatically setting the `old_index_count` would give you incorrect results.